src/app/data-planes/views/DataPlaneListView.spec.ts

```diff
@@ -112,10 +88,18 @@ describe('DataPlaneListView', () => {
     expect(tableRows.length).toBe(1)
 
     const firstTableRowHtml = tableRows[0].html()
-    const expectedTableRowStrings = ['cluster-1.gateway-01', 'default', 'BUILTIN', 'gateway']
+    const expectedTableRowStrings = ['transmitter-gateway-0', 'BUILTIN']
     for (const string of expectedTableRowStrings) {
       expect(firstTableRowHtml).toContain(string)
     }
+
+    await dataPlaneTypeFilter.setValue('Delegated')
+    await flushPromises()
+
+    const firstTableRowHtml2 = tableRows[0].html()
+    for (const string of ['transmitter-gateway-0', 'BUILTIN']) {
+      expect(firstTableRowHtml2).toContain(string)
+    }
```

Comment:
Did I read this right, you've set the filter value to `Delegated` but the column value for that row is `BUILTIN`?

Reply:
Good catch, I’ve copied this and didn’t update it and it passed because I didn’t retrieve a new table rows variable so I was asserting against the old markup. Fixed now.